addons/fcs/functions/fnc_onForceUpdate.sqf

```diff
@@ -1,3 +1,18 @@
+/*
+ * Author: ACE
+ *
+ *
+ * Arguments:
+ * Nothing
```

Comment:
This should be `None`, I noticed it was wrong in some other files before as well.